.github/workflows/docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and deploy sphinx docs
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - 'stable/**'
+
+jobs:
+  build_and_deploy_docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox
+          wget https://github.com/jgm/pandoc/releases/download/2.17.1.1/pandoc-2.17.1.1-1-amd64.deb
+          sudo dpkg -i pandoc-2.17.1.1-1-amd64.deb
```

Comment:
so, pandoc is only installable like this (pip has outdated version which is not working).
This was latest version of package on their github releases for given architecture.
Maybe I'll move it as variable, but it will anyway stay in here.

Something like:
```yaml
env:
  PANDOC_PACKAGE: pandoc-2.17.1.1-1-amd64.deb
  PANDOC_VERSION: 2.17.1.1

...

          wget https://github.com/jgm/pandoc/releases/download/$PANDOC_VERSION/$PANDOC_PACKAGE
          sudo dpkg -i $PANDOC_PACKAGE
```

Other option would be to parse for latest release and install it. But it will add uncertainty to CI. I still think having it fixed as variable is better option

Reply:
Have you tried just `apt-get install`ing it?  I'm happy to play with this if you want.

Reply:
Many other Qiskit packages have the following line in their workflows:

```
sudo apt-get install -y pandoc graphviz
```

Reply:
You are right, it is a better idea. I will try and let you know